Riot/Modules/Home/AllChats/AllChatsViewController.swift

```diff
@@ -800,290 +800,290 @@ extension AllChatsViewController: BannerPresentationProtocol {
+//            MXLog.warning("[AllChatsViewController] showSettingsSecurityScreen: cannot instantiate SettingsViewController")
+//            return
+//        }
+//
+//        guard let securityViewController = SecurityViewController.instantiate(withMatrixSession: session) else {
+//            MXLog.warning("[AllChatsViewController] showSettingsSecurityScreen: cannot instantiate SecurityViewController")
+//            return
+//        }
+//
+//        settingsViewController.loadViewIfNeeded()
+//        AppDelegate.theDelegate().restoreInitialDisplay {
+//            self.navigationController?.viewControllers = [self, settingsViewController, securityViewController]
+//        }
+//    }
+//
+//    private func showOnboardingFlowAndResetSessionFlags(_ resetSessionFlags: Bool) {
+//        // Check whether an authentication screen is not already shown or preparing
+//        guard self.onboardingCoordinatorBridgePresenter == nil && !self.isOnboardingCoordinatorPreparing else {
+//            return
+//        }
+//
+//        self.isOnboardingCoordinatorPreparing = true
+//        self.isOnboardingInProgress = true
+//
+//        if resetSessionFlags {
+//            resetReviewSessionsFlags()
+//        }
+//
+//        AppDelegate.theDelegate().restoreInitialDisplay {
+//            self.presentOnboardingFlow()
+//        }
+//    }
+//
+//    private func resetReviewSessionsFlags() {
+//        reviewSessionAlertHasBeenDisplayed = false
+//        RiotSettings.shared.hideVerifyThisSessionAlert = false
+//        RiotSettings.shared.hideReviewSessionsAlert = false
+//    }
+//
+//    private func presentOnboardingFlow() {
+//        MXLog.debug("[AllChatsViewController] presentOnboardingFlow")
+//
+//        let onboardingCoordinatorBridgePresenter = OnboardingCoordinatorBridgePresenter()
+//        onboardingCoordinatorBridgePresenter.completion = { [weak self] in
+//            guard let self = self else { return }
+//
+//            self.onboardingCoordinatorBridgePresenter?.dismiss(animated: true, completion: {
+//                self.onboardingCoordinatorBridgePresenter = nil
+//            })
+//
+//            self.isOnboardingInProgress = false   // Must be set before calling didCompleteAuthentication
+//            self.allChatsDelegate?.allChatsViewControllerDidCompleteAuthentication(self)
+//        }
+//
+//        onboardingCoordinatorBridgePresenter.present(from: self, animated: true)
+//        self.onboardingCoordinatorBridgePresenter = onboardingCoordinatorBridgePresenter
+//        self.isOnboardingCoordinatorPreparing = false
+//    }
+//
+//    private func refreshSelectedControllerSelectedCellIfNeeded() {
+//        guard splitViewController != nil else {
+//            return
+//        }
+//
+//        // Refresh selected cell without scrolling the selected cell (We suppose it's visible here)
+//        self.refreshCurrentSelectedCell(false)
+//    }
+//}
```

Comment:
It is commented because we don't have the same onboarding on Tchap, and because this is in a class we don't use in Tchap currently.
We should fix this when we will use the new app layout (by implementing the Tchap onboarding on this part).